### DISCUSSION TITLE
it might be time for a new desktop release

### DISCUSSION THREAD

**Discussion by Rubiidev-IPad** [cytrinc/bark-coding#220] [New Features (not block requests)]
what do you think?

**Comment by raynepaws** [comment]
i think that's a good idea, since adding users is a pretty big change.

moved to "new features"

**Comment by mariocraft987** [comment]
yea

**Comment by nmsderp** [comment]
https://github.com/mariocraft987/bark-coding/releases/tag/0.0.18 done
